addons/website_event_jitsi/__manifest__.py

```diff
@@ -10,11 +10,9 @@
     'website': 'https://www.odoo.com/app/events',
     'depends': [
         'website_event',
-        'website_jitsi',
```

Comment:
Any reason to remove this one ? Imo I would still keep it, we just need to not auto install

Reply:
This module is supposed to be just an option in event to activate/desactivate jitsi, therefore I thinks we should not have it in the dependencies right ?

Reply:
TODO: Check with LNA